website/docs/faqs/Troubleshooting/ide-session-unknown-error.md

```diff
@@ -0,0 +1,20 @@
+---
+title: I'm receiving an 'Your IDE session experienced an unknown error and was terminated. Please contact support'.
+description: "Add a repository when seeing IDE unknown error"
+sidebar_label: 'Receiving unknown error in the IDE'
+id: IDE
+
+---
+
+If you're experiencing the below error when you try to open the cloud IDE, it may be due to a few scenarios but it typically indicates a missing repository.
```

Comment:
sugg:
- iirc we prefer to use "the following" (vs "below") for accessibility 
- "may" should be used to mean _permission_ like `you may sit down`. "may" has several meanings depending on context so this convention is for translation purposes

```suggestion
If you're seeing the following error when you launch the dbt Cloud IDE, it could be due to a few scenarios but, commonly, it indicates a missing repository:
```